doc/source/performance/threadpool.rst

```diff
@@ -4,20 +4,20 @@
  Thread Pool
 =============
 
-|MySQL| executes statements using one thread per client connection. Once the number of connections increases past a certain point performance will degrade. 
+|MySQL| executes statements using one thread per client connection. Once the number of connections increases past a certain point performance will degrade.
```

Comment:
Perhaps line wrap if this is touched anyway?

Reply:
Sure.